packages/oruga-next/src/components/dropdown/Dropdown.vue

```diff
@@ -223,6 +223,18 @@ const props = defineProps({
         type: [String, Array, Function] as PropType<ComponentClass>,
         default: undefined,
     },
+    positionClass: {
```

Comment:
Please add jsdoc descriptions to all new classes.

Reply:
oops. added.